configure.ac

```diff
@@ -273,10 +273,6 @@ AC_ARG_ENABLE([unix-lib],
   [AS_HELP_STRING([--disable-unix-lib],
     [do not build the unix library])])
 
-AC_ARG_ENABLE([bigarray-lib],
-  [AS_HELP_STRING([--disable-bigarray-lib],
-    [do not build the legacy separate bigarray library])])
```

Comment:
I believe you might be asked to keep the option around and make it error gracefully instead.
e.g. https://github.com/ocaml/ocaml/pull/10893#issuecomment-1012276828

Reply:
Thank you 🙂

Reply:
Thanks, fixed